src/retry-queue.js

```diff
@@ -6,41 +6,29 @@ export class RetryQueue extends RequestQueueScaffold {
         super()
         this.captureMetrics = captureMetrics
         this.isPolling = false
-        this._requestRetriesMap = {} // <RequestId, number>
-        this._counterToQueueMap = {}
-        this._pollerCounter = 1
-        this._areWeOnline = 'onLine' in window.navigator ? window.navigator.onLine : true
-        this._offlineBacklog = []
+        this.queue = []
+        this.areWeOnline = true
 
         if ('onLine' in window.navigator) {
+            this.areWeOnline = window.navigator.onLine
             window.addEventListener('online', () => {
-                this._handleWeAreNowOnline()
```

Comment:
the reason for extracting this code was for test purposes where mocking online/offline event listeners would be messy and instead we can just call this